compiler/damlc/daml-preprocessor/src/DA/Daml/Preprocessor.hs

```diff
@@ -91,6 +91,7 @@ importDamlPreprocessor = fmap onModule
     where
         onModule y = y {
           GHC.hsmodImports =
+            newImport True "GHC.Types" :
```

Comment:
To add some context this is because the template desugaring from https://github.com/digital-asset/ghc/pull/47 uses `primitive`.